client/components/location-selector.js

```diff
@@ -1,29 +1,27 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import { useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
-
 import Field from './field';
 import getLocations from '../helpers/get-locations';
 
```

Comment:
This did have some behaviour changes initially but they were reverted - decided to keep the refactor to using hooks.